nil/tests/common.go

```diff
@@ -153,18 +153,10 @@ func WaitZerostate(t *testing.T, client client.Client, shardId types.ShardId) {
 func WaitShardTick(t *testing.T, client client.Client, shardId types.ShardId) {
 	t.Helper()
 
-	block, err := client.GetBlock(t.Context(), shardId, "latest", false)
-	require.NoError(t, err)
-
-	require.Eventuallyf(
-		t,
-		func() bool {
-			block, err := client.GetBlock(t.Context(), shardId, transport.BlockNumber(block.Number+1), false)
-			return err == nil && block != nil
-		},
-		ShardTickWaitTimeout,
-		ShardTickPollInterval,
-		"Failed to wait for shard %d tick after block %d", shardId, block.Number)
+	require.Eventually(t, func() bool {
```

Comment:
Now this code checks that at least one block is returned. Actually it can be zerostate block. It doesn't wait for block ticks. In case if consensus fails this function passes.

Reply:
For network startup it's probably ok. But in case you need to be sure that some transaction doesn't break validators  after execution it won't work anymore

Reply:
But ok. It's probably not required for now.